.github/workflows/master.yaml

```diff
@@ -39,20 +39,17 @@ jobs:
         run: |
           minikube start --wait all
           minikube docker-env | sed '/^#/d;s/="\(.*\)"/=\1/g;s/export //g' >> $GITHUB_ENV
-      - name: Cache Go Modules
+      - name: Go Cache
         uses: actions/cache@v1
         with:
-          path: ~/go/pkg/mod
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            ~/Library/Caches/go-build
+            %LocalAppData%\go-build
```

Comment:
I wonder if we should add a step that outputs `go env GOMODCACHE` and `go env GOCACHE` and reference them here instead. It's a little confusing to see 3 directories that are just the OS variants of `GOCACHE`.